Framework/CurveFitting/src/Algorithms/DoublePulseFit.cpp

```diff
@@ -195,8 +195,8 @@ void DoublePulseFit::execConcrete() {
     doublePulseFunction = getDoublePulseFunction(function, pulseOffset, firstPulseWeight, secondPulseWeight);
   }
 
-  auto newFunc = Mantid::API::FunctionFactory::Instance().createInitialized(doublePulseFunction->asString());
-  auto newFunc1 = Mantid::API::FunctionFactory::Instance().createInitialized(
+  Mantid::API::FunctionFactory::Instance().createInitialized(doublePulseFunction->asString());
```

Comment:
Similarly here, are functions created here used at all later on? If yes, how would they be referenced...?

Reply:
Same, I removed these two lines.